src/Symfony/Component/DependencyInjection/ParameterBag/ParameterBag.php

```diff
@@ -264,6 +219,9 @@ public function escapeValue($value)
         return $value;
     }
 
+    /**
+     * {@inheritdoc}
+     */
     public function unescapeValue($value)
```

Comment:
Yes it is, after resolving a value we may want to unescape it.
An example if I use
```php
$resolver = new ParameterResolver(function() { return 'foobar'; });
$resolver->resolveValue('foo %bar% %%');
```
I will obtain ``foo foobar %%``.